apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -57,9 +176,12 @@
 .recorder-controls {
   display: flex;
 
-  button {
-    flex: 1;
-    margin: 0 10px;
+  .btn {
+    width: 100%;
+
+    & + .btn {
```

Comment:
Unnecessary parent selector (&)

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#unnecessaryparentreference)